mozilla_repo_urls/errors.py

```diff
@@ -8,8 +8,13 @@ def __init__(self, url_string) -> None:
 
 
 class UnsupportedPlatformError(RepoUrlsBaseError):
+    @property
+    def supported_platforms(self):
+        return list(self._supported_platforms)
```

Comment:
Question: I'm sorry, I'm not sure to understand. What is the need for a property here? `supported_platforms`/`supported_hosts` is already a list and we could make it a public attribute, I think.

Reply:
Removed